Energy Web/energy-web-mainnet-starter/README.md

```diff
@@ -0,0 +1,90 @@
+# SubQuery - Example Project for Energy Web
+
+[SubQuery](https://subquery.network) is a fast, flexible, and reliable open-source data indexer that provides you with custom APIs for your web3 project across all of our supported networks. To learn about how to get started with SubQuery, [visit our docs](https://academy.subquery.network).
+
+**This SubQuery project indexes all transfers and approval events for the  on Energy Web**
```

Comment:
@bgodlin this did not correctly format

Reply:
This SubQuery project indexes all transfers and approval events for the {{token_name}} (`{{token address}}`) on {{network name}} {{environment}}

Reply:
Actually, suggest we define 

```
const prettyName = `${name}${environment.toLower() === 'mainnet' ? '' : ' ' + environment}`;
```

And then

This SubQuery project indexes all transfers and approval events for the {{token_name}} ({{token address}}) on {{prettyName}}